tools/proteore_expression_rnaseq_abbased/add_expression_data.xml

```diff
@@ -0,0 +1,177 @@
+<tool id="rna_abbased_data" name="Add expression data" version="2020.04.06">
+<description> (RNAseq or Immuno-assays)[Human Protein Atlas]
+</description>
+<requirements>
+  <requirement type="package" version="3.6.1">r-base</requirement>
+</requirements>
+<stdio>
```

Comment:
remove and use the detect_errors= attribute https://docs.galaxyproject.org/en/latest/dev/schema.html#error-detection